### PR TITLE
DEVOPS-671 Adding bors settings and CI configs

### PR DESCRIPTION
**Pull request by jorge-cr-13**
Creating the bors main configuration file and adapting the CI files for accepting bors commands.

### CODE REVIEW

File: .github/workflows/bors_ci.yml [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+  DART_VERSION: '2.14.4'
+  DART_WORKSPACE: ${{ github.workspace }}/discovery_engine
+  RUST_WORKSPACE: ${{ github.workspace }}/discovery_engine_core
+  RUST_NIGHTLY: nightly-2021-09-09
+  RUSTFLAGS: "-D warnings"
+  RUSTDOCFLAGS: "-D warnings"
+
+
+jobs:
+  selection:
```
Comment (acrrd) [inline .github/workflows/bors_ci.yml:30]: Can't we call the other one like we do in `xayn_ai`?

Comment (jorge-cr-13) [reply .github/workflows/bors_ci.yml:30]: I'm not sure if you are refering to the name of the CI? I changed the name of them to match with the ones with `xayn_ai`

Comment (acrrd) [reply .github/workflows/bors_ci.yml:30]: Sorry I meant if we can call `ci.yml` and not duplicating it here.

Comment (jorge-cr-13) [reply .github/workflows/bors_ci.yml:30]: Ah ok. Yes, I can create workflow reuse.

**Pull request closed without merging**